templates/homepage/index.html.twig

```diff
@@ -0,0 +1,78 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Танцы Беларусі{% endblock %}
```

Comment:
Create translations files

Reply:
I don't understand what to do.

Reply:
translations into which languages?

Reply:
Do not use russian text in twig. Use translations